apl-db-updater/src/main/resources/db/migration/h2/apl/V1_0__Initialize_tables.sql

```diff
@@ -1039,15 +993,14 @@ CREATE TABLE IF NOT EXISTS `shard`
 
 CREATE TABLE IF NOT EXISTS `shard_recovery`
 (
-    `shard_recovery_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
-    `state`             varchar(150) COLLATE utf8_bin NOT NULL,
-    `object_name`       varchar(500) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-    `column_name`       varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+    `shard_recovery_id` bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key,
+    `state`             varchar(150)  NOT NULL,
```

Comment:
**shard_recovery_id_state_object_idx** will not work on MariaDb because of removed 'COLLATE utf8_bin'